package.json

```diff
@@ -82,8 +85,11 @@
     "url": "https://github.com/ckeditor/ckeditor5-vue.git"
   },
   "files": [
+    "scripts/postinstall.js",
     "dist",
-    "scripts/postinstall.js"
+    "README.md",
+    "CHANGELOG.md",
```

Comment:
Previously, we did not publish the `CHANGELOG.md` - see https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code. Is it needed on npm?

Reply:
I don't have a strong opinion about it, but we publish it for all other packages. So I guess it would be good to keep it consistent and publish it.